src/hb-face.cc

```diff
@@ -372,11 +384,13 @@ hb_face_reference_table (const hb_face_t *face,
 
 /**
  * hb_face_reference_blob:
- * @face: a face.
- *
+ * @face: A face object
  *
+ * Fetches a pointer to the binary blob that contains the
+ * specified face. Can only be used on faces created with
+ * hb_face_builder_create().
```

Comment:
I’m not sure the last sentence is correct. This function is basically `hb_face_reference_table(face, HB_TAG_NONE)`, so it should work similar to my comment above.